jonathanmusila/Quesioner-React
# Initialise React-App

#### What does this PR do?
- creates the initial react app using babel and webpack

#### Description of Task to be completed?
- [x] install babel and presets
- [x] create a webpack.config.js file
- [x] create a .babelrc file

#### How should this be manually tested?
- ensure that you have installed node and node package manager(npm) in your machine
- `git clone` `https://github.com/jonathanmusila/Quesioner-React.git`
- run the command `npm install` to install dependancies
- run command `npm start` to fire up the project

#### Any background context you want to provide?
None
